content/docs/1.6.0/advanced-resources/security/volume-encryption.md

```diff
@@ -3,9 +3,7 @@ title: Volume Encryption
 weight: 2
 ---
 
-Longhorn supports encrypted volumes by utilizing the linux kernel module `dm_crypt` via `cryptsetup` for the encryption.
-Further we use the Kubernetes secret mechanism for key storage, which can be further encrypted and guarded via appropriate permissions.
-An encrypted volume results in your data being encrypted while in transit as well as at rest, this also means that any backups taken from that volume are also encrypted.
+A volume in `Filesystem` or `Block` mode is supported to be encrypted by utilizing the Linux kernel module `dm_crypt` via `cryptsetup`. Further we use the Kubernetes secret mechanism for key storage, which can be further encrypted and guarded via appropriate permissions. An encrypted volume results in your data being encrypted while in transit as well as at rest, this also means that any backups taken from that volume are also encrypted.
```

Comment:
I reorganized and reworded the paragraph. Please check if the technical details are correct.

Longhorn supports volume encryption at the storage class level, which means that volumes in both `Filesystem` and `Block` mode can be encrypted while in transit and at rest. Moreover, backups taken from encrypted volumes are also encrypted. This is all accomplished through the Linux kernel module `dm_crypt`, the command-line utility `cryptsetup`, and Kubernetes Secrets. `dm_crypt` and `cryptsetup` handle the creation and management of encrypted devices, while Secrets (and related permissions) facilitate secure storage of encryption keys.